pallets/xyk/src/tests.rs

```diff
@@ -1314,6 +1315,174 @@ fn liquidity_rewards_deactivate_W() {
 	});
 }
 
+#[test]
+#[serial]
+fn liquidity_rewards_calculate_work_after_deactivate() {
```

Comment:
is that test related to the impl from this pr ?

Reply:
tests are related to some scenarios which were addressed in this and previous pr